pydm/widgets/slider.py

```diff
@@ -78,9 +78,9 @@ def mouseMoveEvent(self, event):
             delta = event.pos() - self.dragStartPos
 
             if self.orientation() == Qt.Horizontal:
-                delta_value = (delta.x() / self.getSliderLength()) * (self.maximum() - self.minimum())
+                delta_value = int(round((delta.x() / self.getSliderLength()) * (self.maximum() - self.minimum())))
```

Comment:
do we need both int cast and round?

Reply:
It looks like not, I thought round would return a float if given a float, but it looks like without a ndigits passed it returns an int. I will remove the int casting